package.json

```diff
@@ -17,7 +18,7 @@
     "typecheck": "tsc --noEmit",
     "lint": "eslint .",
     "lint:fix": "eslint . --fix",
-    "build": "tsc --declaration",
+    "build": "tsc -p tsconfig.dist.json && mkdir -p dist/codegen && cp -r codegen/src dist/codegen/src",
```

Comment:
is the cp required - can we just tell tsconfig to emit there? and maybe it will make the dir for us idk

Reply:
The `cp` is about copying the codegen-related `.pkl` files used by `pkl-gen-typescript`.

We could also use eg. https://www.npmjs.com/package/copyfiles if we didn't want to rely on `mkdir` and `cp` from the host.

Reply:
oh right, im dumb. looks good then :)